client/store/util.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a string representation of a sorted query object.
+ *
+ * @format
+ * @param {Object} query Current state
+ * @return {String}       Query Key
+ */
+
+export function getQueryKey( query = {} ) {
```

Comment:
I'm not sure if `getQueryKey` is the best name here since we are not getting a key but a string. I have a similar function in my WIP branch that I called serializeQuery -- but getQueryString might work too.

The `* @format` should also be separate at the top of the file. Looks like the formatter might have put it in the docblock :).

Reply:
`getQueryString` is a better name for `stringifyQuery`, so it might be confusing to have both. Maybe `getJsonString`, to indicate exactly what it does?